test/api/tracing_ready_test.rb

```diff
@@ -33,13 +33,17 @@
   end
 
   it 'default_test_solarwinds_ready' do
+    skip if ENV['APM_RUBY_TEST_STAGING_KEY'].to_s.empty?
```

Comment:
I think this should be an Error instead of skip when required env var for running test is not set--if we somehow accidentally remove this GH Action secret we'd never notice. In upcoming [update of CONTRIB docs](https://swicloud.atlassian.net/browse/NH-116007) we should note the required env vars when running tests locally.

Reply:
that seems acceptable to me

Reply:
Updated the rake test